ServerWhitelist.yml

```diff
@@ -1947,19 +1947,15 @@ AllowedPrefixes:
     - https://cdn.discordapp.com/attachments/1063514401375780874/1198357469903204383/SexLab_Framework_PPLUS_-_V2.3.2.1_-_1.5.97.zip?ex=65da4bdd&is=65c7d6dd&hm=3121fe0ab2594a7a661e07a0d19c0dafe7f72813946e2be02479e04977e51a9b& # Sexlab + 2.3.2.1 / SE 1.5.9.7 ( Inline Permission from Scrab )
     - http://www.mediafire.com/file/o5yacfkx21p3b8i/Sexlab_DLL_16640_Replacer.rar # Sexlab Framework 16640 Working DLL ( Inline Permission from Scrab )
     - https://cdn.discordapp.com/attachments/1177560192192352336/1178355556113731604/SexLab_P_Quick_Scene_Start_Spell_-_4.0.0.zip?ex=65db5ba1&is=65c8e6a1&hm=0afebe8fd8aedc5ce84171cc1b0d649fd62bbe888f4d75c89f95fcdb6c34ab4c& # Sexlab + Quick Spell Inline
```

Comment:
Permission from MissCorruption. Not sure if you need a Screenshot for this.

Reply:
https://github.com/wabbajack-tools/opt-out-lists/pull/816/files/0cd523ba0c98ed2760278ad256229320b5791670..922e596e07136e17782ef0bb9aff8db73489ee55#r1501290676